students/andykwon/lesson02/gridprinter2.py

```diff
@@ -0,0 +1,22 @@
+def print_grid2(num1, num2):
```

Comment:
Nice grid!
Some feedback:
- If you like the extra newlines for readability, that's fine, but note that that amount of space isn't necessary. Whatever you prefer
- num1 and num2 aren't the best parameter names. Something a little more descriptive would be ideal. It'd help with readability.
- For both gridprinter2 and gridprinter, you have twice as many spaces between the dashes as specified. Not a huge issue, just a heads up. 
- You can make this a little more simple if you use string multiplication instead of for loops in some of these cases. Loops can be computationally expensive, so for example instead of:
```
for j in range (num1):
    print(end='+' + ' - ' * num2)
```
you can do something like:
```
print(("+" + " - " * num2) * num1 + "+")
```